speckle/objects/base.py

```diff
@@ -45,28 +42,18 @@ def __setattr__(self, name: str, value: Any) -> None:
             attr.__set__(self, value)
         super().__setattr__(name, value)
 
-    def validate_prop_name(self, name: str) -> Tuple[bool, SpeckleException]:
+    @classmethod
+    def validate_prop_name(cls, name: str) -> None:
```

Comment:
this method was purposely not raising the value error and passing that responsibility on as that is how it was recently implemented in sharp. i presumed there was a reason for this, but haven't actually checked so will ask about this and report back

Reply:
checked w @AlanRynne and we're good to go on this refactor 👍